docs/data/material/integrations/interoperability/interoperability.md

```diff
@@ -600,7 +600,7 @@ If you use a different framework, or already have set up your project, follow th
  };
 ```
 
-3. Add the `important` option, using the id of your app wrapper. For example, `#__next` for Next.js and `"#root"` for CRA:
+3. Add the `important` option, using the id of your app wrapper. For example, `#__next` for Next.js:
```

Comment:
We say `#__next` here, but the code snippet below shows `#root`.

Reply:
I removed the appearance of CRA because, since CRA is taken off, pointing to it might not be necessary